include/slang/util/String.h

```diff
@@ -80,20 +80,9 @@ inline char charToLower(char c) {
 int editDistance(std::string_view left, std::string_view right, bool allowReplacements = true,
                  int maxDistance = 0);
 
-#if defined(_WIN32)
-
 /// Gets a string representation of the given path, in UTF-8 encoding.
 inline std::string getU8Str(const std::filesystem::path& path) {
-    return path.string();
-}
-
-#else
-
-/// Gets a string representation of the given path, in UTF-8 encoding.
-inline const std::string& getU8Str(const std::filesystem::path& path) {
-    return path.native();
+    return path.generic_string();
```

Comment:
I think you should keep the ifdef -- for non-Windows builds the native() method already returns the right thing and returns by reference, so we save an allocation + copy of the string. Changing the WIN32 side of the branch to use generic_string instead seems fine though.